CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 ### Fixed
+- Terminate gracefully on SIGTERM - @recmo
+- Use exec in Dockerfile - @recmo
```

Comment:
Thanks for pointing it out. I didn't feel good about occupying two changelog lines for something so trivial.